### ISSUE TITLE
bauh review in Germany

### ISSUE THREAD

**Issue by AWHubGit** [vinifmor/bauh#199]
Do you know this:

[bauh review](https://www.heise.de/ratgeber/Linux-Softwarepakete-verwalten-mit-Bauh-6185724.html)

**Comment by ranplayer** [comment]
No, I didn't. Thanks for sharing @AWHubGit . Good to know that bauh has been useful for people.

**Comment by vinifmor** [comment]
No, I didn't. Thanks for sharing this @AWHubGit . Good to know that bauh has been useful for people.

**Comment by vinifmor** [comment]
@tuxfanmatze , all the AppImage database is  automatically generated by a server-side script that consumes data from AppImageHub. If this application is not available, it may be related to missing data related to the download links.

**Comment by tuxfanmatze** [comment]
@vinifmor,  I would appreciate if you can check why the automated Nextcloud client appimage retrieval from flatkap repo is not working. Appimage is the official current Nextcloud client for Linux.
Especially in the open source/Linux world Nextcloud is now a growing important cloud and a good replacement for the big player clouds: Google Drive, Microsft Onedrive, Dropbox etc.

**Comment by vinifmor** [comment]
@tuxfanmatze , I have manually added the missing links and files to backend script. It will be available on the database tomorrow.

I'm going to create a specific issue template for suggesting AppImage applications not listed on the database.

![bauh-nextcloud](https://user-images.githubusercontent.com/4312414/142731209-8bbb5c69-048c-41d0-b953-1bf0706e4877.png)

**Comment by vinifmor** [comment]
I've just updated the issue templates to include one for AppImage suggestions: https://github.com/vinifmor/bauh/issues/new/choose

**Comment by tuxfanmatze** [comment]
**Thanks for your effort and constant development!** In the beginning, Nextcloud was only briefly found in new Nextcloud versions as AppImage. (In the meantime there was no Nextcloud entry in app.txt anymore). Now it seems to be stable and the Nextcloud AppImage entry is constantly present.
Otherwise, the app works fine, even in each new version, currently bauh 0.9.25 and Linux Mint 20.3.
